prov/efa/src/efa_av.c

```diff
@@ -377,88 +522,17 @@ int efa_rdm_av_insert_one(struct efa_av *av, struct efa_ep_addr *addr,
 		goto out;
```

Comment:
This seems to return 0 (which seems to indicate that there is no problem) and not the fi_addr_t (as indicated in the comment above).